Plugins/GitUIPluginInterfaces/RepositoryHosts/GitProtocol.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+
+namespace GitUIPluginInterfaces.RepositoryHosts
+{
+    public static class GitProtocol
+    {
+        public const string Ssh = "SSH";
+        public const string Https = "HTTPS";
+
+        public static bool IsUrlUsingProtocol(this string url, string protocol)
+        {
+            if (string.IsNullOrWhiteSpace(protocol) || string.IsNullOrWhiteSpace(url))
+            {
+                return false;
+            }
+
+            return url.StartsWith($"{protocol}://", StringComparison.CurrentCultureIgnoreCase);
```

Comment:
>ssh protocol do not require ssh: prefix

I know. I wanted to keep it simple for the moment but I was wrong. Fixed.

>Also, local paths shouls be considered for reference repos

I thought it could be added afterward when required (because I don't know exactly the need)

>However, I do not see that this PR limits the functionality compared to now, it is possible to enhance later.

That was the goal initialy but I derived a little adding this ssh method...